nilearn/surface/surface.py

```diff
@@ -1975,3 +1992,139 @@ def new_img_like(ref_img, data):
         mesh=copy.deepcopy(mesh),
         data=data,
     )
+
+
+
+def threshold_img(
```

Comment:
A lot of this is duplicated from nilearn.image.threshold_img in terms of code and doc string.

Probably another candidate function may be able to handle both niimg and surfimg.